org-cyf-itp/content/structuring-data/sprints/2/day-plan/index.md

```diff
@@ -5,35 +5,38 @@ emoji= '🧑🏾‍🤝‍🧑🏾'
 menu_level = ['sprint']
 weight = 3
 [[blocks]]
-name="Energiser"
-src="blocks/energiser"
+name="Energiser: Word Chain"
+src="energisers/word-chain"
+time=20
 [[blocks]]
 name="Morning orientation"
 src="blocks/morning-orientation"
 time=15
 [[blocks]]
-name="Workshop"
-src="blocks/workshop"
-time=140
-  [[blocks.nested.blocks]]
-    name="Questions and Reviews [PD] (60 Mins)"
-    src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/questions-and-reviews"
-    time=0
-  [[blocks.nested.blocks]]
-    name="Debugging [Tech] (60 Mins)"
-    src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/debugging"
-    time=0
-  [[blocks.nested.blocks]]
-    name="Javascript Test Your Understanding Level 2 [Tech] (30 Mins)"
-    src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/js1-week-2"
-    time=0
+name="Questions and Review Workshop"
+src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/questions-and-reviews"
+time=60
+[[blocks]]
+name="Morning break"
+src="blocks/morning-break"
+[[blocks]]
+name="Time Traveller"
+src="energisers/time-traveller"
+time=65
 [[blocks]]
 name="lunch"
 src="blocks/lunch"
 [[blocks]]
+name="Stand Up"
+src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/stand-up"
```

Comment:
I tweaked this a bit in https://github.com/CodeYourFuture/CYF-Workshops/pull/284